src/gui/windows/download_controller_mappings.zig

```diff
@@ -16,16 +16,21 @@ const HorizontalList = @import("../components/HorizontalList.zig");
 
 pub var window = GuiWindow {
 	.contentSize = Vec2f{128, 64},
-	.pos = Vec2f{0, 0},
-	.showTitleBar = false,
+	.showTitleBar = true,
 	.hasBackground = true,
-	.closeable = false
+	.closeable = false,
+	.relativePosition = .{
+		.{
+			.ratio = 1.0,
+		},
+		.{
+			.ratio = 1.0
+		}
+	}
```

Comment:
The intended way to move things into one corner would actually be
```
	.relativePosition = .{
		.{ .attachedToFrame = .{.selfAttachmentPoint = .upper, .otherAttachmentPoint = .upper} },
		.{ .attachedToFrame = .{.selfAttachmentPoint = .upper, .otherAttachmentPoint = .upper} },
	},
```
Also please add the trailing commas in multi-line initializations, this makes it easier to move around or add more attributes below.